css-view-transitions-2/Overview.bs

```diff
@@ -1237,25 +1295,86 @@ When [[css-view-transitions-1#setup-transition-pseudo-elements-algorithm|setting
 
 		1. Append |group| to |parentGroup|.
 
-		1. When setting the animation keyframes given |transform|, [=multiply=] |transform| by the inverse matrix of |groupContainerElement|'s [=old transform=].
+		1. When setting the animation keyframes given |transform|, [=multiply=] |transform| by the inverse matrix of (|groupContainerElement|'s [=old transform=] · |groupContainerElement|'s [=old nested group transform=]).
 
 	Note: It is TBD how this is resolved when the old and new groups mismatch. See <a href="https://github.com/w3c/csswg-drafts/issues/10631">Issue 10631</a>.
 </div>
 
 ### Adjust the group's 'transform' to be relative to its containing ''::view-transition-group()'' ### {#vt-group-transform-adjust}
 <div algorithm="additional pseudo-element style update steps (group)">
-When [[css-view-transitions-1#style-transition-pseudo-elements-algorithm|updating the style of the transition pseudo-element]], perform the following steps before setting the [=captured element/group styles rule=], given a [=captured element=] |capturedElement|, given a |transform| and a {{ViewTransition}} |transition|:
+When [[css-view-transitions-1#style-transition-pseudo-elements-algorithm|updating the style of the transition pseudo-element]], perform the following steps before setting the [=captured element/group styles rule=], given a [=captured element=] |capturedElement|, a |transform|, and a {{ViewTransition}} |transition|:
 
 	1. Set |capturedElement|'s [=transform from snapshot containing block=] to |transform|.
 
 	1. If |capturedElement|'s [=containing group name=] is not null, then:
 
-		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][|capturedElement|'s [=containing group name=]].
+		1. Let |groupContainerElement| be |transition|'s [=ViewTransition/named elements=][|capturedElement|'s [=containing group name=].
 
-		1. [=Multiply=] |transform| by the inverse matrix of |groupContainerElement|'s [=transform from snapshot containing block=].
+		1. [=Multiply=] |transform| by the inverse matrix of (|groupContainerElement|'s [=transform from snapshot containing block=] · |groupContainerElement|'s [=new nested group transform=]).
 </div>
 
+## Capturing layered CSS properties ## {#layered-capture-algorithms}
+
+### Compute the layered-capture style ### {#layered-capture-compute-style}
+
+<div algorithm="compute layered capture style">
+To compute the <dfn>layered capture style</dfn> of an {{Element}} |element|:
+
+	1. Let |styles| be a « ».
+	1. For each |property| in the list of [=layered capture properties=], [=list/append=] the [=string/concatenate|concatentation=] of « |property|, `":"`, |element|'s [=computed value=] of |property|, `";"` » to |styles|.
+	1. Return the [=string/concatenate|concatentation=] of |styles|.
+</div>
 
+### Capture the old layered properties ### {#capture-new-layered-props-algorithm}
```

Comment:
We need to clearly define a consistent size for this snapshot. L1 defines it as the border box bounds with any painted content outside of it as ink overflow. This helps because irrespective of how big the snapshot is (based on overflow computed later in the pipeline), css properties which apply on top of image bounds (like `object-view-box`) behave consistently.

Also helps to not have to compute the snapshot size for layout.

And for layered capture, the snapshot size should probably always be content box size as far as web observable effects or layout is concerned. Since the goal is to capture the content and similar to flat capture, we can ensure that the content box origin in the image aligns with the content box origin of replaced old/new image pseudo.

Reply:
Done